wp-admin/js/widgets/media-image-widget.js

```diff
@@ -39,15 +39,20 @@
 
 			props = {
 				attachment_id: attachment.id,
-				url: attachment.sizes[ displaySettings.size ].url,
-				size: displaySettings.size,
-				width: 0, // Reset.
-				height: 0, // Reset.
 				align: displaySettings.align,
-				caption: attachment.caption,
 				alt: attachment.alt,
+				caption: attachment.caption,
+				image_classes: '',
+				image_title: '',
+				link_classes: '',
+				link_rel: '',
+				link_url: displaySettings.linkUrl,
+				link_target_blank: false,
```

Comment:
I don't know, it just came completely unexpected. I associate the modal settings with the image, not the widget. The `title` for example is something that I'm sure doesn't carry over from one image to the other, much like custom css classes.

I'd expect the settings to be like they are when first setting up the widget and selecting an image